### ISSUE TITLE
not work on ubuntu 22.04 

### ISSUE THREAD

**Issue by ARm1110** [v2fly/fhs-install-v2ray#258]
cmd
 bash install-release.sh
output:
info: No new version. The current version of V2Ray is  .

**Comment by mahdizojaji** [comment]
i got this error in ubuntu 22.04:
`Check failed! Please check your network or try again.`

**Comment by LILERF** [comment]
> i got this error in ubuntu 22.04: `Check failed! Please check your network or try again.`

same here!

```
Downloading verification file for V2Ray archive: https://github.com/v2fly/v2ray-core/releases/download/v5.2.1/v2ray-linux-64.zip.dgst
error: Check failed! Please check your network or try again.
removed: /tmp/tmp.7hYSyHE2v4
```

**Comment by mahdizojaji** [comment]
> > i got this error in ubuntu 22.04: `Check failed! Please check your network or try again.`
> 
> delete these lines[304-312]
> 
> ```
>  # Verification of V2Ray archive 
>  for  LISTSUM  in  ' md5 '  ' sha1 '  ' sha256 '  ' sha512 ' ;  die 
>  SUM="$(${LISTSUM}sum "$ZIP_FILE" | sed 's/ .*//')" 
>  CHECKSUM="$(grep ${LISTSUM^^} "$ZIP_FILE".dgst | grep "$SUM" -o -a | uniq)" 
>  if [[ "$SUM" != "$CHECKSUM" ]]; then 
>  echo 'error: Check failed! Please check your network or try again.' 
>  return 1 
>  fi 
>  done 
> ```
> 
> then:
> 
> bash -x ./install-release.sh --version v5.2.1

I do that and script ignore thats eror but when use v2ray command i got error v2ray command not found.
actually v2ray does not installed completely.

**Comment by IceCodeNew** [comment]
the original issue does not seem to be the same with later comments. @ARm1110 if you are still experiencing problem, pls open another issue.
For the following comments, the bug is fixed in #263 and the hack is no longer needed.
